index.html

```diff
@@ -0,0 +1,484 @@
+                  <td class="f-bold f-client">$49.99<span class="f-price"> / Month</span></td>
+                </tr>
+                <tr>
+                  <td>UNLIMITED Projects</td>
+                </tr>
+                <tr>
+                  <td>UNLIMITED Users</td>
+                </tr>
+                <tr>
+                  <td>1000 Gb user</td>
+                </tr>
+                <tr>
+                  <td>UNLIMITED Calendar</td>
+                </tr>
+              </tbody>
+              <tfoot>
+                <tr>
+                  <td class="td-btn"><a href=""><button class="btn f-bold">SIGN UP</button></a></td>
+                </tr>
+              </tfoot>
+            </table>
+          </div>
+        </section>
+      </article>
+      <article class="b-white" id="portfolio">
+        <h2 class="f-bold f-large">PORTFOLIO</h2>
+        <img src="/img/Slider.png" alt="Crispers - Nice, crispy and creative landing page" class="l-slider slider-break">
+        <img src="/img/Slider_mini.png" alt="Crispers - Nice, crispy and creative landing page" class="l-slider slider">
+        <ul class="f-bold l-subnav">
+          <li class="s-portfolio-filter-links l-subnav-link">All Works</li>
+          <li class="s-portfolio-filter-links l-subnav-link">Webdesign</li>
+          <li class="s-portfolio-filter-links l-subnav-link">UI design</li>
+          <li class="s-portfolio-filter-links l-subnav-link">Icons design</li>
+          <li class="s-portfolio-filter-links l-subnav-link">Brand & Logotypes</li>
+        </ul>
+        <div class="l-flex-container l-portfolio-flex">
+          <figure class="l-flex-item f-small-line-height">
+            <div class="first-level-hover-wrapper">
+             <div class="item-wrapper">
+                <div class="s-portfolio-image">
+                  <img src="/img/youTube_iphone_app.png" alt="YouTube Iphone App Example" class="s-portfolio-image">
+                 <span class="portfolio-overlay-filter"> </span>
+                 <div class="portfolio-overlay-button"><a href="">+</a></div>
+               </div>
+             </div>
+           </div>
+            <figcaption>
+              <p><strong class="f-bold">Youtube iphone app</strong></p>
+              <p>UI design, brand & logotypes</p>
+            </figcaption>
+          </figure>
+          <figure class="l-flex-item f-small-line-height">
+            <div class="first-level-hover-wrapper">
+              <div class="item-wrapper">
+                <div class="s-portfolio-image">
+                  <img src="/img/hexagon_one_page.png" alt="Web Design Example">
+                 <span class="portfolio-overlay-filter"> </span>
+                 <div class="portfolio-overlay-button"><a href="">+</a></div>
+               </div>
+             </div>
+           </div>
+            <figcaption>
+              <p><strong class="f-bold">Hexagon onepage</strong></p>
+              <p>Webdesign, brand & logotypes</p>
```

Comment:
One more ampersand. This'll be my last comment on it -- there are more in your file, so keep an eye out for those as well.